prediction/src/algorithms/segment/src/data_generation.py

```diff
@@ -0,0 +1,78 @@
+import glob
+import os
+
+import numpy as np
+import pylidc as pl
+
+
+def get_dicom_paths(in_docker=True):
+    """Return DICOM paths to all LIDC directories depending on whether Docker is used or not
+    e.g. ['../images_full/LIDC-IDRI-0001/1.3.6.1.4.1.14519.5.2.1.6279.6001.298806137288633453246975630178/' \
+          '1.3.6.1.4.1.14519.5.2.1.6279.6001.179049373636438705059720603192']
+
+    Args:
+        in_docker: whether this method is invoked from within docker or from the prediction directory
+    """
+    if in_docker:
+        paths = glob.glob(os.path.join('/images_full', 'LIDC-IDRI-*', '**', '**'))
+    else:
+        paths = glob.glob(
+            os.path.join(os.getcwd(), 'src', 'tests', 'assets', 'test_image_data', 'full', 'LIDC-IDRI-*', '**', '**'))
+    return paths
+
+
+def get_assets_dir():
+    return os.path.abspath(os.path.join(os.getcwd(), 'src', 'algorithms', 'segment', 'assets'))
```

Comment:
This is surely just the same as underscore underscore file underscore underscore ?

Reply:
I also see some other uses of ``../images`` still lingering around? Not sure if an oversight, or...?

Reply:
Do the constants introduced in 773df14 make sense?